.github/dockerized-gh_actions_runners/Kubernetes/github-actions-windows-deployment.yml

```diff
@@ -0,0 +1,28 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: iis
```

Comment:
Please include more descriptive names/metadata in order to know more about the deployment and pods. Such as: iis-github-actions-runners or something like that. Thanks!

Reply:
Of course, I renamed tags in the last commit, thx.